backend/src/api/v1/activities/services/weather_match_service.py

```
@@ -0,0 +1,346 @@
+            elif weather.temperature > 25:
+                context_tips.append("🧊 아이스 음료와 에어컨 자리 추천")
+        elif activity_type == "park":
+            if 18 <= weather.temperature <= 25 and weather.wind_speed <= 5:
+                context_tips.append("🌸 산책, 피크닉, 야외 운동에 완벽한 조건")
+        
+        return " | ".join(context_tips) if context_tips else "현재 날씨에서 방문 가능한 장소입니다."
+    
+    def get_optimal_time_recommendation(self, weather: WeatherEntity, place: PlaceEntity) -> str:
+        """최적 방문 시간을 추천합니다."""
+        activity_type = self._normalize_place_type(place.type)
+        current_hour = datetime.now().hour
```

Ruff (0.12.2)

Local variable `current_hour` is assigned to but never used

Remove assignment to unused variable `current_hour`

(F841)